osp/serializers/value.py

```diff
@@ -1,121 +1,125 @@
-from rest_framework import serializers
-
 from osp.models import (
-    Answer, ChoiceValue, CheckboxValue, DropdownValue,
-    ParagraphValue, ShortAnswerValue, DateValue, TimeValue, FileUploadValue
+    CheckboxValue,
+    ChoiceValue,
+    DateValue,
+    DropdownValue,
+    FileUploadValue,
+    ParagraphValue,
+    ShortAnswerValue,
+    TimeValue,
 )
 from osp.serializers.answer import AnswerReadSerializer, AnswerWriteSerializer
 
+
 # read serializers
 class ChoiceValueReadSerializer(AnswerReadSerializer):
-
     class Meta:
 
         model = ChoiceValue
-        fields = AnswerReadSerializer.Meta.fields + ['value']
+        fields = AnswerReadSerializer.Meta.fields + ["value"]
 
-class CheckboxValueReadSerializer(AnswerReadSerializer):
 
+class CheckboxValueReadSerializer(AnswerReadSerializer):
     class Meta:
 
         model = CheckboxValue
-        fields = AnswerReadSerializer.Meta.fields + ['value']
+        fields = AnswerReadSerializer.Meta.fields + ["value"]
 
-class DropdownValueReadSerializer(AnswerReadSerializer):
 
+class DropdownValueReadSerializer(AnswerReadSerializer):
     class Meta:
 
         model = DropdownValue
-        fields = AnswerReadSerializer.Meta.fields + ['value']
+        fields = AnswerReadSerializer.Meta.fields + ["value"]
 
-class ParagraphValueReadSerializer(AnswerReadSerializer):
 
+class ParagraphValueReadSerializer(AnswerReadSerializer):
     class Meta:
 
         model = ParagraphValue
-        fields = AnswerReadSerializer.Meta.fields + ['value']
+        fields = AnswerReadSerializer.Meta.fields + ["value"]
 
-class ShortAnswerValueReadSerializer(AnswerReadSerializer):
 
+class ShortAnswerValueReadSerializer(AnswerReadSerializer):
     class Meta:
 
         model = ShortAnswerValue
-        fields = AnswerReadSerializer.Meta.fields + ['value']
+        fields = AnswerReadSerializer.Meta.fields + ["value"]
 
-class DateValueReadSerializer(AnswerReadSerializer):
 
+class DateValueReadSerializer(AnswerReadSerializer):
     class Meta:
 
         model = DateValue
-        fields = AnswerReadSerializer.Meta.fields + ['value']
+        fields = AnswerReadSerializer.Meta.fields + ["value"]
 
-class TimeValueReadSerializer(AnswerReadSerializer):
 
+class TimeValueReadSerializer(AnswerReadSerializer):
     class Meta:
```

Comment:
You can inherit AnswerReadSerializer.Meta here

Reply:
I think not to change it because it can be extended in the future.